packages/workers-shared/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+	root: true,
+	extends: ["@cloudflare/eslint-config-worker"],
+	ignorePatterns: ["dist"],
```

Comment:
The `eslint-config-worker` package already has comprehensive ignore patterns.

https://github.com/cloudflare/workers-sdk/blob/a6a773dd7f88a6091e6e0f6715eda1a43c374332/packages/eslint-config-worker/index.js#L8-L14

Was there a reason for overriding here?